lib/cre8or.ts

```diff
@@ -24,3 +27,21 @@ export const getLockedCount = async (address: string) => {
 
   return count
 }
+
+export const transferCre8orToSmartWallet = async (
```

Comment:
can we make this a more generic function?
- send more than just a `cre8or` (any `ERC721`)
- send anywhere. not just to a smart wallet.

```suggestion
export const transferErc721 = async (
```

Reply:
made hook 🎣